collect-settings.rst

```diff
@@ -97,16 +97,14 @@ Form Management Settings
 
 - :guilabel:`Default to finalized` When enabled, forms are automatically finalized upon reaching the end of the form. You can opt out of finalizing any form when completing it.
 - :guilabel:`Constraint processing` used to select when response constraints are validated: After each swipe, or at finalization.
-- :guilabel:`High res video` enables high-resolution recordings.
-- :guilabel:`Image size` is used to set an image size for all the widgets.There are five size options available as shown below:
-
-.. image:: /img/collect-settings/image-settings.png
-  :alt: Image settings 
-  :class: device-screen-vertical
-
-.. versionadded:: 1.11.0*
-   :guilabel:`Image size` feature will be added in *Collect v1.11.0* (upcoming release).
-  
+- :guilabel:`High res video` enable or disable high-resolution video recordings if supported by the video application used.
```

Comment:
.. to enable or disable   or  ..enables or disables.

Reply:
👍